scripts/akrtma3d/exakrtma3d_obsprep_cloud.ksh

```diff
@@ -76,7 +76,7 @@ postmsg "$jlogfile" "$msg"
 
 targetfile="NASALaRCCloudInGSI.bufr"
 if [ -f ${DATA}/${targetfile} ] ; then
-  cpreq ${DATA}/${targetfile} ${COMINobsproc_rtma3d}/rtma.t${cyc}z.${targetfile}
+  cpreq ${DATA}/${targetfile} ${COMINobsproc_rtma3d}/${NET}.t${cyc}z.${targetfile}
```

Comment:
I believe this should be ${RUN} as this is the output file from the lightning code and dependent on the grid.